ensembl_genes/Red.py

```diff
@@ -0,0 +1,264 @@
+""" Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+    Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License. """
+
+import eHive
+
+import os
+import filecmp
+import errno
+import subprocess
+
+import sqlalchemy as db
+import sqlalchemy_utils as db_utils
+
+# sqlalchemy requires MySQLdb but MySQLdb doesn't support Python 3.x
+# pymysql can be imported and used instead
+import pymysql
+pymysql.install_as_MySQLdb()
```

Comment:
I don't know but I needed to make it work at least locally.

Reply:
Have you tried this: `mysql+pymysql://<username>:<password>@<host>/<dbname>`?

Reply:
I think Jorge's test is more about testing the ORM part of the pymysql library, which I avoided because I think that could be useful if we decided to centralise a definition of the core database schema to be shared across Ensembl but not for individual packages as it would require to redefine the schema every time if I understood correctly how it works. For this pilot project I thought it was not necessary.

I'll try using "mysql+pymysql" instead of "mysql" as driver in the string for the db connection when calling "create_engine". I'm guessing the effect will be similar but we'll be able to remove the "install_as_MySQLdb() call" if that helps with readability.

Reply:
It didn't work. It failed when calling "create_engine":
sqlalchemy.exc.NoSuchModuleError: Can't load plugin: sqlalchemy.dialects:mysql.pymsql

So I undid the change and kept the "install_as_MySQLdb()" as it works.

Reply:
Ok, I think we are missing something but if this works then let's use that. Maybe it's the name of the function, `install_as_MySQLdb`, which bother me